rescheduling-service/src/routes/index.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import availableAppointmentsRouter from './availableAppointments';
+import availablePatientsRouter from './availablePatients';
+import createOfferRouter from './createOffer';
+
+const router = express.Router();
+
+router.use('/available-appointments', availableAppointmentsRouter);
+router.use('/patients', availablePatientsRouter);
+router.use('/create-offer', createOfferRouter);
```

Comment:
Also, if you add a controller folder, you can delete those routes and replace them with instances of controller classes, so it will be easier to understand what they are doing.
And you'll not need an extra folder for each route.